impl/client/src/main/javascript/web/pentaho/data/filter/standard.js

```diff
@@ -22,11 +22,15 @@ define([
   "./not",
   "./isEqual",
   "./isIn",
+  "./isGreater",
+  "./isLess",
+  "./isGreaterOrEqual",
+  "./isLessOrEqual",
   "./true",
   "./false"
 ], function(
-    abstractFactory, treeFactory, propFactory, andFactory, orFactory, notFactory, isEqFactory, isInFactory,
-    trueFactory, falseFactory) {
+    abstractFactory, treeFactory, propFactory,
+    andFactory, orFactory, notFactory, isEqFactory, isInFactory, isGtFactory, isLtFactory, isGteFactory, isLteFactory, trueFactory, falseFactory) {
```

Comment:
This is used in this file alone and was named in line with the existing `isEqFactory`. The names used elsewhere do not use abbreviations. If you feel it is important I can rename all factories in this file, new and old, to not use abbreviations either.